### PR TITLE
Wes july13

### PR DESCRIPTION
**Pull request by wgarlock**

### CODE REVIEW

File: shuup_vendor_reviews/models.py [hunk 1]
```diff
@@ -21,6 +48,18 @@ def for_reviewer(self, shop, reviewer):
     def approved(self):
         return self.filter(status=ReviewStatus.APPROVED)
 
+    def for_reviewer_dict_options(self, shop, reviewer):
+        reviews = dict()
+        suppliers = Supplier.objects.filter(supplier_reviews__reviewer=reviewer)
```
Comment (tulimaki) [inline shuup_vendor_reviews/models.py:53]: Filter out suppliers that are disabled.

File: shuup_vendor_reviews/dashboard_items.py [hunk 2]
```diff
@@ -35,3 +36,25 @@ def get_context(self):
             self.request.person
         ).order_by("-created_on")[:5]
         return context
+
+
+class VendorReviewWithOptionsDashboardItem(DashboardItem):
```
Comment (tulimaki) [inline shuup_vendor_reviews/dashboard_items.py:41]: Are reqular vendor reviews now disabledwhen this is on. Or does that have separate setting?

File: shuup_vendor_reviews/views.py [hunk 3]
```diff
@@ -104,3 +106,153 @@ def get_reviews_page(self):
...
+                defaults=dict(
+                    rating=data["rating"],
+                    comment=data["comment"],
+                    would_recommend=data["would_recommend"],
+                    status=ReviewStatus.PENDING,
+                )
+            )
+
+
+class VendorReviewGroupForm(FormGroup):
```
Comment (tulimaki) [inline shuup_vendor_reviews/views.py:165]: Instead of for group maybe REST API endpoint would be easier option?

Comment (tulimaki) [reply shuup_vendor_reviews/views.py:165]: Do not change this if this actually works already.

File: shuup_vendor_reviews/dashboard_items.py (hunk 2 above)
Comment (wgarlock) [reply shuup_vendor_reviews/dashboard_items.py:41]: They have separate setting. They can be, both on, both off, or either on/off

File: shuup_vendor_reviews/views.py (hunk 3 above)
Comment (wgarlock) [reply shuup_vendor_reviews/views.py:165]: Yeah this is working now. I couldn't think of a cleaner way of implementing this as I was working on it.

**Comment by tulimaki** [comment]
Replaced by https://github.com/shuup/shuup-product-reviews/pull/35

**Pull request closed without merging**